lib/generators/rolemodel/source_map/source_map_generator.rb

```diff
@@ -0,0 +1,11 @@
+#frozen_string_literal: true
+
+module Rolemodel
+  class ReadmeGenerator < Rails::Generators::Base
```

Comment:
Shouldn't the class name be `SourceMapGenerator`?

Reply:
It should.
Resolved.

Reply:
Looks like this isn't resolved

Reply:
Resolved.